### ISSUE TITLE
Bug: "Sweep Conversations" Appears Even without texter-feedback enabled; might just be a confusing name

### ISSUE THREAD

**Issue by mayefsky** [ProgressiveCoders/Spoke#1945]
**Describe the bug**
The "sweep conversations" link on the view campaign admin screen shows even without `texter-feedback` enabled; it also shows from Message Review when looking at a single texter's messages. I think this might actually be intentional--I have a hard time understanding from the code whether "Sweep conversations" is supposed to be a review activity even without text-feedback enabled or not.
If Sweep Conversations is intended to simply be a way for an admins to review the conversations of a particular texter, that's a great feature, but I might suggest a different name--Sweep (to me, and to at least one of my users, which is how this came up) sounds like an action that move/changes/destroys, or defragements, or something. If it's just to review conversations maybe the links can just say Review Conversations?

**To Reproduce**
1. Go to the admin campaign page for a live page (without texter-feedback enables)
2."Sweep conversations" link is available for individual texters, leading to some confusion

**Comment by Frydafly** [comment]
Thanks @mayefsky 
I'm adding the documentation label since I think we need to have some clear documentation on the function(s) of texter feedback & sweep conversations

**Comment by codygordon** [comment]
Yes, this isn't a bug and the link should appear even if Texter Feedback isn't enabled. It is as you described, a way to simply view all the texter's conversations.

I also do agree "Review conversations" might be a better more generic description. @ryanjcrump are you good with that change?

**Comment by Frydafly** [comment]
Thanks @codygordon! Are you considering making a language change to this feature?

**Comment by Frydafly** [comment]
I've kept the documentation tag since it would be nice to get clear docs on how sweep-conversations works!

**Comment by ryanjcrump** [comment]
Yes, I support a name change. Ultimately, I would love it if any combination of Message Review filters where viewable as a list view (as current in Message Review) or as a Conversation View (as the current texter-feedback sweeping view) 

So, I propose that the language in this link says "Conversation View" or "Texter View"
